wpeverest/everest-forms
# Feature - Payment improvement

### All Submissions:






### Changes proposed in this Pull Request:

This PR is about the quantity field and email triggers to user conditionally if the payment is completed or failed.

Closes # .

### How to test the changes in this Pull Request:

1. Create a form or edit the existing forms and activate the payment addons.
2. Go to the settings tab and email and enable conditional logic and choose payment on the select field section and set the rule as per your desire.
3. Check the Email according to the condition.

### Types of changes:




### Other information:




### Changelog entry

> Feature - Quantity field introduced.
> Feature - Added conditional logic on the Email Section to send the email according to payment status.
